tailwind.config.js

```diff
@@ -1,8 +1,8 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [],
+  content: ["./src/**/*.{js,ts,jsx,tsx,html}"],
```

Comment:
The content should be filled to tell where the styles will be applied by using tailwind.